modules/virt-about-workload-security.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-additional-security-privileges-controller-and-launcher.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-workload-security_{context}"]
+= About workload security
+
+By default, virtual machine (VM) workloads do not run with root privileges in {VirtProductName}.
+
+For each VM, a `virt-launcher` pod runs an instance of `libvirt` to manage the VM process. The `virt-launcher` pod runs in _session mode_, meaning that all users in the `virt-launcher` container must have a nonzero user identifier (UID). Therefore, VMs run as unprivileged pods, adhering to the security principle of least privilege.
```

Comment:
Session mode is a libvirt concept/mode. It tells libvirt to run VMs under the same user as the client who requests the VM and will not do privileged operations.

Reply:
Thank you for this clarification! I'll tweak this paragraph to reflect that.